# Translations template for ceilometer.
# Copyright (C) 2016 ORGANIZATION
# This file is distributed under the same license as the ceilometer project.
# FIRST AUTHOR <EMAIL@ADDRESS>, 2016.
#
#, fuzzy
msgid ""
msgstr ""
"Project-Id-Version: ceilometer 6.0.0.0b4.dev6\n"
"Report-Msgid-Bugs-To: EMAIL@ADDRESS\n"
"POT-Creation-Date: 2016-03-07 06:08+0000\n"
"PO-Revision-Date: YEAR-MO-DA HO:MI+ZONE\n"
"Last-Translator: FULL NAME <EMAIL@ADDRESS>\n"
"Language-Team: LANGUAGE <LL@li.org>\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=utf-8\n"
"Content-Transfer-Encoding: 8bit\n"
"Generated-By: Babel 2.2.0\n"

#: ceilometer/collector.py:176
#, python-format
msgid "Dispatcher failed to handle the %s, requeue it."
msgstr ""

#: ceilometer/coordination.py:50
msgid ""
"Group ID: %{group_id}s, Members: %{members}s, Me: %{me}s: Current agent "
"is not part of group and cannot take tasks"
msgstr ""

#: ceilometer/coordination.py:88 ceilometer/coordination.py:100
msgid "Error connecting to coordination backend."
msgstr ""

#: ceilometer/coordination.py:115
msgid "Error sending a heartbeat to coordination backend."
msgstr ""

#: ceilometer/coordination.py:146
#, python-format
msgid "Error joining partitioning group %s, re-trying"
msgstr ""

#: ceilometer/coordination.py:198
msgid "Error getting group membership info from coordination backend."
msgstr ""

#: ceilometer/service_base.py:118
#, python-format
msgid "Unable to load changed pipeline: %s"
msgstr ""

#: ceilometer/service_base.py:140
#, python-format
msgid "Unable to load changed event pipeline: %s"
msgstr ""

#: ceilometer/agent/manager.py:467
#, python-format
msgid "Skipping %(name)s, keystone issue: %(exc)s"
msgstr ""

#: ceilometer/agent/plugin_base.py:109
msgid "Fail to process notification"
msgstr ""

#: ceilometer/api/hooks.py:65
#, python-format
msgid "Failed to connect to db, purpose %(purpose)s retry later: %(err)s"
msgstr ""

#: ceilometer/compute/pollsters/memory.py:108
#, python-format
msgid "Could not get Resident Memory Usage for %(id)s: %(e)s"
msgstr ""

#: ceilometer/dispatcher/database.py:54
#, python-format
msgid "Failed to connect to db, purpose %(purpose)s re-try later: %(err)s"
msgstr ""

#: ceilometer/dispatcher/database.py:97
#, python-format
msgid "Failed to record metering data: %s"
msgstr ""

#: ceilometer/dispatcher/database.py:128
#, python-format
msgid "Error processing event and it will be dropped: %s"
msgstr ""

#: ceilometer/dispatcher/gnocchi.py:102
#, python-format
msgid "Required field %s not specified"
msgstr ""

#: ceilometer/dispatcher/gnocchi.py:105
#, python-format
msgid "Required field %(field)s should be a %(type)s"
msgstr ""

#: ceilometer/dispatcher/gnocchi.py:234
msgid "Failed to connect to Gnocchi."
msgstr ""

#: ceilometer/dispatcher/gnocchi.py:250
#, python-format
msgid "Failed to load resource due to error %s"
msgstr ""

#: ceilometer/dispatcher/http.py:126
msgid "Status Code: %{code}s. Failed todispatch event: %{event}s"
msgstr ""

#: ceilometer/event/endpoint.py:69
msgid "Fail to process a notification"
msgstr ""

#: ceilometer/event/storage/impl_elasticsearch.py:110
#: ceilometer/event/storage/impl_hbase.py:121
#: ceilometer/event/storage/impl_sqlalchemy.py:203
#: ceilometer/event/storage/impl_sqlalchemy.py:205
#: ceilometer/event/storage/pymongo_base.py:69
#, python-format
msgid "Failed to record event: %s"
msgstr ""

#: ceilometer/hardware/pollsters/generic.py:69
#, python-format
msgid "Missing field %s"
msgstr ""

#: ceilometer/hardware/pollsters/generic.py:72
#, python-format
msgid "Unrecognized type value %s"
msgstr ""

#: ceilometer/hardware/pollsters/generic.py:107
msgid "Passed resource dict must contain keys resource_id and resource_url."
msgstr ""

#: ceilometer/hardware/pollsters/generic.py:126
#, python-format
msgid "Cannot load inspector %(name)s: %(err)s"
msgstr ""

#: ceilometer/hardware/pollsters/generic.py:149
#, python-format
msgid "Skip invalid resource %s"
msgstr ""

#: ceilometer/hardware/pollsters/generic.py:177
#, python-format
msgid "inspector call failed for %(ident)s host %(host)s: %(err)s"
msgstr ""

#: ceilometer/hardware/pollsters/generic.py:227
#: ceilometer/meter/notifications.py:197
#, python-format
msgid "Error loading meter definition : %(err)s"
msgstr ""

#: ceilometer/meter/notifications.py:65
#, python-format
msgid "Required fields %s not specified"
msgstr ""

#: ceilometer/meter/notifications.py:74
#, python-format
msgid "Invalid type %s specified"
msgstr ""

#: ceilometer/publisher/kafka_broker.py:83
#, python-format
msgid "Failed to connect to Kafka service: %s"
msgstr ""

#: ceilometer/publisher/messaging.py:168
msgid "Failed to retry to send sample data with max_retry times"
msgstr ""

